src/components/countries-list/CountriesList.vue

```diff
@@ -22,15 +22,19 @@ export default {
         code: {
             type: String,
             default: 'alpha3'
-        }
+        },
+        url: {
+            type: String,
+            default: 'https://cdn.jsdelivr.net/npm/world_countries_lists@latest/data/countries/en/countries.json'
+        },
```

Comment:
@osamaamer2024 I see that you went with the `url` as a prop, and not the language, is there another API URL that will have the same response structure of this one, which is also related to countries?